environment.yml

```diff
@@ -7,14 +7,16 @@ dependencies:
   - numpy
   - pip
   - isce2>=2.6.3
-  - dem_stitcher>=2.5.8
   - rasterio
   - shapely
+  - pyproj
```

Comment:
Why was `pyproj` added as a dependency? None of the new code does anything projection related.

Reply:
It was a forgotten import that may have been a dependency of another package.

Reply:
https://github.com/ASFHyP3/hyp3-isce2/blob/develop/src/hyp3_isce2/packaging.py#L14 :+1: